src/date-time/format-fuzzy-date.js

```diff
@@ -0,0 +1,85 @@
+import processPattern from '../util/process-pattern.js';
+
+const msPerDay = 86400000;
+
+const thirtySeconds = 30000;
+const ninetySeconds = 90000;
+
+const oneMinute = 60000;
+const twoAndAHalfMinutes = 150000;
+const fortyFiveMinutes = 2700000;
+
+const oneHour = 3600000;
+const oneHourThirtyMinutes = 5400000;
+const twoHoursThirtyMinutes = 9000000;
+
+export default function FuzzyDateFormatter(localeData, locale, timeZone) {
+	this.format = localeData.date.formats.fuzzyFormats;
+	this.locale = locale;
+	this.timeZone = timeZone ||  new Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
ie 11 doesn't resolve this.  However using an undefined timezone for intl.DateTimeFormat in ie 11 behaves like it uses the existing client timezone.  Ideally we'd always be passing in a configured zone